macstadium-orka-server/src/main/java/com/macstadium/orka/client/OrkaClient.java

```diff
@@ -171,6 +172,15 @@ private String executeCall(Request request) throws IOException {
 
     @Override
     public void close() throws IOException {
-        this.delete(this.endpoint + TOKEN_PATH, "");
+        String response = this.get(this.endpoint + HEALTH_PATH);
+        Gson gson = new Gson();
+        JsonObject jsonObject = gson.fromJson(response, JsonObject.class);
+        String apiVersionString = jsonObject.get("api_version").toString();
```

Comment:
This shouldn't be `toString`. It will return "a string within the string". That is ""2.2.0"" instead of "2.2.0". The correct method is `getAsString`